frontend/package.json

```diff
@@ -6,13 +6,29 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "algoliasearch": "^3.35.1",
+    "feather-icons": "^4.28.0",
+    "framer-motion": "^2.9.4",
+    "instantsearch.css": "^7.4.5",
     "react": "^16.14.0",
+    "react-anchor-link-smooth-scroll": "^1.0.12",
     "react-dom": "^16.14.0",
-    "react-scripts": "3.4.3"
+    "react-instantsearch-dom": "^6.8.2",
+    "react-modal": "^3.11.2",
+    "react-rnd": "^10.2.3",
+    "react-router-dom": "^5.2.0",
+    "react-scripts": "^4.0.0",
+    "react-slick": "^0.27.12",
+    "slick-carousel": "^1.8.1",
+    "styled-components": "^5.2.0",
+    "tailwindcss": "^1.9.6",
+    "twin.macro": "^1.11.0",
+    "use-in-view": "^1.0.15"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "BROWSER='firefox developer edition' react-scripts start",
```

Comment:
Are you sure it should be a part of the source code? What if other developer don't have this browser (like me)? Maybe there is another way to specify it locally?

Reply:
I am not sure how to do this. I don't have any experience rebasing or reverting. Can you give me a link on how what to do? 

> You need to base your branch on top of the latest `main`. Currently your branch is stale, and that is why the CI checks have not been run — they are defined in the commit that is not on your branch.
> ![image](https://user-images.githubusercontent.com/6065071/97181893-63343600-17ce-11eb-9171-1da20fe54b12.png)
> 
> Your branch has conflicts with the main branch, particularly in the README file. I suggest you to revert your changes in that file because it almost duplicate what is already added to it in the `main` branch. You can add frontend-specific details into its own README file inside the `frontend` directory.
> 
> There is no need to include “Trello card task” into pull request title and commit messages. Please read the README file in the repository after you rebase your branch on top of `main`.